src/components/ProjectList/ProjectItem.js

```diff
@@ -1,15 +1,14 @@
-/* eslint-disable */
+/* @flow */
```

Comment:
Just a bit of background for you: So originally we had `/* @flow */` in all files, but hardly used flow... So flow had a lot of errors. Now we try to be strict, so when we have a flow annotation in the top of the file, we actually use flow. 

For components this means annotation prop types using flow. I would recommend you to do that. As an example I've done this here: https://github.com/Unity-Technologies/Tanto/blob/master/src/components/PullRequestSummary/PullRequestSummary.js#L71